ci/libs/utils.sh

```diff
@@ -0,0 +1,170 @@
+#!/bin/bash
+
+. ci/libs/variables.sh
+
+cleanup(){
+	rm -rf $TMP_DIR/vol*
+	rm -rf $MNT_DIR/logs
+	docker stop $(docker ps -aq)
```

Comment:
we should either have a list of docker id's that we started to remove, removing all docker is a bad idea

Reply:
In order to start the integration test I think we need a clean environment